service/internal/access/v2/errors.go

```diff
@@ -0,0 +1,22 @@
+package access
+
+import "errors"
+
+var (
+	ErrMissingRequiredSDK                          = errors.New("access: missing required SDK")
```

Comment:
Maybe don't group these error values all together in one place abstracted from their use? If you do want to create something like this, only put ones that are actionable by other packages here, and only export ones that need to be exported.

These mostly seem very arbitrary

Reply:
This is a good suggestion. I will break them out rather than grouping together. I think I started the file when we only had `pdp.go` and just appended from there, which doesn't make very friendly DX.